release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -170,6 +170,116 @@ See xref:../web_console/creating-quick-start-tutorials.adoc#creating-quick-start
 
 The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#displaying-the-insights-status-in-the-web-console_using-insights-to-identify-issues-with-your-cluster[Insights plug-in] is now integrated into the {product-title} web console. Insights provides cluster health data, such as the number of total issues and total risks of the issues. Risks are labeled as *Critical*, *Important*, *Moderate*, or *Low*. You can quickly navigate to {cloud-redhat-com} for further details about the issues and how to fix them.
 
+[id="ocp-4-7-ibm-z"]
+==== IBM Z and LinuxONE
+
+With this release, IBM Z and LinuxONE are now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE] or xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network] for installation instructions.
+
+[discrete]
+===== Notable Enhancements
+
+The following new features are supported on IBM Z and LinuxONE with {product-title} {product-version}:
+
+* KVM on {op-system-base} 8.3 or later is supported as a hypervisor for user-provisioned installation of {product-title} {product-version} on IBM Z and LinuxONE.  See xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE] for installation instructions.
+* Multipathing
+* OpenShift Pipelines TP
+* Red Hat OpenShift Service Mesh
+* OVN-Kubernetes with an initial installation of {product-title} 4.7
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* z/VM Emulated FBA devices on SCSI disks
+
+[discrete]
+===== Supported features
+
+The following features are also supported on IBM Z and LinuxONE:
+
+* CodeReady Workspaces
+* OpenShift Do (odo)
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Log forwarding
+** Precision Time Protocol (PTP) hardware
+** CSI volume snapshots
+
+* The following {product-title} features are unsupported:
+** OpenShift Virtualization
+** CodeReady Containers (CRC)
```

Comment:
Not sure if we want to mention this here. Published separately on the Customer portal and not part of OCP docs. Though related to OCP, cause the product helps you run OCP locally.

Reply:
If they are supported, no matter where the docs are published, we should mention them.